robotd/visiond/token_display.py

```diff
@@ -0,0 +1,54 @@
+from PIL import Image, ImageDraw
+import numpy as np
+from PIL import ImageFont
+
+from visiond.tokens import Token
+
+
+def _draw_centered(pos, text, font, color, dr):
+    w, h = dr.textsize(text, font=font)
+    x, y = pos[0] - w / 2, pos[1] - h / 2
+    dr.text((x+2,y), text, font=font, fill='black')
+    dr.text((x-2,y), text, font=font, fill='black')
+    dr.text((x,y+2), text, font=font, fill='black')
+    dr.text((x,y-2), text, font=font, fill='black')
+    dr.text((x,y), text, font=font, fill=color)
+
+
+def display_tokens(tokens: Token, image: Image):
+    new_image = image.convert('RGBA')
+    dr = ImageDraw.Draw(new_image)
+    fnt = ImageFont.load_default()
+    font_size = 28
+    try:
+        # Pretty font, pretty sure I'm the only one to have it though - Andy
+        fnt = ImageFont.truetype("DejaVuSans.ttf", font_size, encoding="unic")
+    except:
+        try:
+            # More common font
+            fnt = ImageFont.truetype("arial.ttf", font_size, encoding="unic")
+        except:
+            print("fallback to default font")
```

Comment:
This message isn't massively actionable—could do with mentioning what fonts you _did_ look for here and where.